enterprise_subsidy/apps/subsidy/models.py

```diff
@@ -44,9 +44,9 @@ class SubsidyReferenceChoices:
     object that caused the subsidy to come into existence.
     """
     # TODO: confirm the actual Salesforce object to use, I sort of just made up "opportunity_product_id"...
-    OPPORTUNITY_PRODUCT_ID = "opportunity_product_id"
+    OPPORTUNITY_LINE_ITEM = "salesforce_OpportunityLineItem"
```

Comment:
nit: we should snake-case this value to make it consistent with how we typically define the persisted value of choices.